.prettierignore

```diff
@@ -0,0 +1,2 @@
+npm install --save-dev --save-exact prettier
```

Comment:
what does this refer to? shouldn't this be ignoring files instead?

Reply:
removed